run-tests.sh

```diff
@@ -22,29 +22,31 @@ TESTS_SSP="stacksmash"
 TESTS_ASAN="stacksmash"
 TESTS_UBSAN="ubsan"
 for arch in $ARCHS; do
-    mkdir -p $arch
+  for target_os in $TARGET_OSES; do
+    TEST_DIR="$arch-$target_os"
```

Comment:
This patch seems incorrectly split. There's no `$TARGET_OSES` available here yet. (The `for target_os ...` line seems incorrectly indented as well.)

I'd prefer to have this commit to be a pure change from `$arch` to `$TEST_DIR`, without anything else involved.

Reply:
Indeed, I fixed the commit